addons/project/models/project_task.py

```diff
@@ -1962,3 +1978,20 @@ def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=Fals
                 group['personal_stage_type_id_count'] = group.pop('personal_stage_type_ids_count', 0)
             return result
         return super().read_group(domain, fields, groupby, offset, limit, orderby, lazy)
+
+    # ---------------------------------------------------
+    # Project Sharing
+    # ---------------------------------------------------
+
+    def project_sharing_toggle_is_follower(self) -> bool:
```

Comment:
It is just typings to get autocompletion, that is, when we will call that method in another method, the IDE will know the value returned is a bool value. Since we don't really use that in the codebase, I think I will remove it. 😅